src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1642,6 +1649,12 @@ public void setUseExistingAccountWithSameEmail(boolean useExistingAccountWithSam
             this.useExistingAccountWithSameEmail = useExistingAccountWithSameEmail;
         }
 
+        public boolean isRedundantFetchAllowed() { return allowSecondFetch; }
+
+        public void setAllowSecondFetch(boolean allowSecondFetch) {
```

Comment:
There are methods in Jenkins that use introspection to identify the getters and setters.  The asymmetry between the name of the getter and the setter in this case will probably confuse anything that uses introspection (like configuration as code and JobDSL)

Since the UI uses `allowSecondFetch`, I think you might consider using `allowSecondFetch` to replace `redundantFetchAllowed`.

```suggestion
        public void setRedundantFetchAllowed(boolean allowSecondFetch) {
```